paint-timing/fcp-only/fcp-bg-image-two-steps.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<head>
+<title>Performance Paint Timing Test: FCP due to background image, in two steps</title>
```

Comment:
What this is really testing is that background image with visibility hidden does not get FCP (while visible does). Perhaps make title more descriptive of this.

Reply:
ok